superchain/configs/sepolia/op.toml

```diff
@@ -10,6 +10,7 @@ canyon_time = 1699981200 # Tue 14 Nov 2023 17:00:00 UTC
 delta_time = 1703203200 # Fri 22 Dec 2023 00:00:00 UTC
 ecotone_time = 1708534800 # Wed 21 Feb 2024 17:00:00 UTC
 fjord_time = 1716998400 # Wed 29 May 2024 16:00:00 UTC
+granite_time = 1723132800 # Thu Aug 8 16:00:00 UTC
```

Comment:
We usually want to give at least 1 week notice for Testnet releases. This would only be 6 days, and it's already Friday and we would still need to pull this into op-geth and op-node, and run on a devnet for a few hours at least, before we roll this out to testnet and make a release.

Reply:
Aug 12 works.

Reply:
updated